setup.cfg

```diff
@@ -122,6 +122,8 @@ install_requires =
     python-dateutil>=2.3, <3
     python-nvd3~=0.15.0
     python-slugify>=3.0.0,<5.0
+    # Required for flask-openID which comes with flask-appbuilder
```

Comment:
```suggestion
    # Required for flask-openID which comes with flask-appbuilder in case of poetry installation
    # earlier versions of the dependency are installed but they do not work for Python 3
    # (pip installs the right version). 
    # More info: https://github.com/apache/airflow/issues/13149#issuecomment-748705193
```